src/components/Fields/TextField.story.js

```diff
@@ -263,6 +263,50 @@ _NB: \`tooltipError\` will ONLY be displayed if \`isValid\` is false._
     ),
 );
 
+/* eslint-disable react/no-multi-comp */
+class ValidationTooltipComponent extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: '',
+            touched: false,
+        };
+    }
+
+    isValid() {
+        const { value, touched } = this.state;
+        if (!touched) {
+            return true;
+        }
+
+        return value !== '';
+    }
+
+    render() {
+        const { value } = this.state;
+        return (
+            <TextField
+                label="Input with a tooltip"
+                isValid={this.isValid()}
+                value={value}
+                onChange={val => this.setState({
+                    touched: true,
+                    value: val,
+                })}
+                tooltipError="Field is required"
+                tooltipPosition={TooltipPosition.TOP_RIGHT}
+            />
+        );
+    }
+}
+
+stories.add(
+    'Validation + Tooltip',
+    () => (
+        <ValidationTooltipComponent />
```

Comment:
The example component in this story does not seem to work as expected – I cannot see any tooltip appear at all.  Can you clarify how this is supposed to work please @diogobeda?

I'd like to move this demo component up into the main tooltips story – I'm happy to do this, but let's get it working first.

Reply:
Hey @mattpauldavies, it appears when the input is `touched` and `empty`. Here's a gif:

![kapture 2018-04-06 at 11 05 05](https://user-images.githubusercontent.com/1817719/38425538-8c3f0e00-398a-11e8-9efd-819e80e1d061.gif)